test/tests/outputs/png/wedge.i

```diff
@@ -17,6 +17,20 @@
   [../]
 []
 
+[AuxVariables]
+  [bnds]
+  []
+[]
+
+[AuxKernels]
+  [bnds]
```

Comment:
I know this is picky by "bnds" is a bad name (even for phase-field it isn't great). Let's go with two_u or something like that.